src/pretix/presale/templates/pretixpresale/event/fragment_subevent_list.html

```diff
@@ -24,7 +24,11 @@
                         {% if subev.best_availability_is_low %}
                             <span class="label label-success-warning">{% trans "Few tickets left" %}</span>
                         {% else %}
-                            <span class="label label-success">{% trans "Book now" %}</span>
+                            {% if event.event.has_paid_item %}
```

Comment:
Müsste es hier nicht auf den `subev` gehen?
```suggestion
                            {% if subev.has_paid_item %}
```

Reply:
ja, da hab ich beim copy pasten geschlafen, habs grad schon selber gepusht. danke!